app/middlewares/validations.js

```diff
@@ -0,0 +1,16 @@
+const { checkSchema, validationResult } = require('express-validator');
+const errors = require('../errors');
+
+const verifySchema = schema => checkSchema(schema);
+
+const validateResult = (req, _, next) => {
+  const err = validationResult(req);
+  if (!err.isEmpty()) {
+    throw errors.schemaValiation(err.mapped());
```

Comment:
Here you should use `return next(error.schema....)`, In middleware/controller functions `next()` should be used instead of throw